src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -51,9 +56,11 @@ public class Swerve extends SubsystemBase {
   Field2d field = new Field2d();
   Vision vision;
 
-  int simGyro = SimDeviceDataJNI.getSimDeviceHandle("navX-Sensor[4]");
+  int simGyro = SimDeviceDataJNI.getSimDeviceHandle(SIM_DEVICE_HANDLE_NAME);
   SimDouble simGyroAngle = new SimDouble(SimDeviceDataJNI.getSimValueHandle(simGyro, "Yaw"));
-  PIDController headingController = new PIDController(5.2, 0, 0.5);
+  PIDController headingController = new PIDController(DEFAULT_PROPORTIONAL_COEFFICIENT,
+                                                      DEFAULT_INTEGRAL_COEFFICIENT,
+                                                      DEFAULT_DERIVATIVE_COEFFICIENT);
```

Comment:
should just be moved to SwerveConstants, and done probably as an array like DRIVE_PID is. `HEADING_PID` would be roughly a suitable name.